js/controller.js

```diff
@@ -58,7 +58,9 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         "countDownFinished": false,
         "countDownCancelled": false,
         "timeToShow": 0,
-        "showing": false
+        "showing": false,
+        "delayedSetEmbedCodeEvent": false,
+        "selectedContentData": null
```

Comment:
Ok, I will rename selectedContentData to delayedContentData and reset it after consumed.